iOS/MobileStarter/react-app/src/HubScreen.scss

```diff
@@ -29,35 +29,32 @@
     }
   }
 
-  .filter-section {
+  .project-source {
     width: 100vw;
     background-color: var(--background-color-2);
-    div {
-      position: relative;
-      padding: 0px 6px 1px 6px;
-      box-sizing: border-box;
-      width: 100vw;
-    }
-    input {
-      font-size: 14px;
-      position: relative;
-      top: -6px;
-      box-sizing: border-box;
-      margin: 0px;
-      width: 100%;
-      height: 38px;
-      padding-left: 13px;
-      padding-right: 13px;
-      background-color: var(--muic-background);
-      color: var(--muic-foreground-2);
-      border: 1px solid var(--muic-gray-9);
-      border-radius: 60px;
-      &:focus {
-        outline: none;
-      }
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    justify-content: center;
+
+    .mui-horizontal-picker {
+      height: 40px;
+      font-size: 18px;
+      margin-left: 8px;
+      margin-right: 8px;
+      margin-bottom: 5px;
     }
   }
 
+  .centered {
+    display: flex;
+    width: 100%;
+    height: 100%;
+    flex-direction: column;
```

Comment:
If all this class does is center a single item, then flex-direction: column shouldn't be needed. If it centers multiple items, then I think perhaps a more descriptive classname is warranted.

Reply:
Yes, in IModelDownloader there are multiple items. I'm open to name suggestions if you have any.

Reply:
How about centered-list?

Reply:
Done